app/uk/gov/hmrc/pushpullnotificationsgateway/connectors/OutboundProxyConnector.scala

```diff
@@ -104,15 +101,16 @@ class OutboundProxyConnector @Inject()(appConfig: AppConfig,
 object OutboundProxyConnector {
   implicit val callbackValidationResponseFormat: OFormat[CallbackValidationResponse] = Json.format[CallbackValidationResponse]
   private[connectors] case class CallbackValidationResponse(challenge: String)
+  val objLogger = Logger("outboundProxyConnector")
```

Comment:
Any reason not to use logger - the object OutboundProxyConnector can extend AppLogger I seem to recall.

Reply:
Have changed this